leo_description/urdf/macros.xacro

```diff
@@ -433,15 +482,19 @@
                        robot_ns:=''">
 
     <xacro:property name="link_prefix" value=""/>
+    <xacro:property name="joint_prefix" value=""/>
     <xacro:if value="${robot_ns != '' and robot_ns != '/'}">
       <xacro:property name="link_prefix" value="${robot_ns}/"/>
+      <xacro:property name="joint_prefix" value="${robot_ns}/"/>
```

Comment:
I'm not convinced the joint names should be prefixed by the robot namespace. I always assumed the joint names are local to each robot as each robot has their own joint_states topic
```suggestion
```